packages/round-manager/src/features/api/payoutStrategy/payoutStrategy.ts

```diff
@@ -63,6 +66,20 @@ export const deployMerklePayoutStrategyContract = async (
   }
 };
 
+/**
+ * @param signerOrProvider
+ * @returns the factory address.
+ */
+export const getDirectPayoutFactoryAddress = async (
+  signerOrProvider: Signer
+): Promise<{ payoutContractAddress: string }> => {
+  const chainId = await signerOrProvider.getChainId();
```

Comment:
Since `signerOrProvider` has no other use here, would it make sense to turn this function into this?

```ts
export const getDirectPayoutFactoryContractAddress = (chainId: ChainId): string => { ... }
```

That way, we turn the function sync, and we avoid it returning something that sounds slightly different that what the name says it would return ("DirectPayoutFactoryAddress" vs "payoutContractAddress").

Reply:
good call :+1: